test/regression/builder/road_network_builder_test.cc

```diff
@@ -587,7 +587,7 @@ std::vector<DirectionUsageTruthTable> InstantiateDirectionUsageBuilderParameters
            {LaneId("1_2_3"), SRange(0., 33.4), "Bidirectional", strict_severity},
            {LaneId("1_2_2"), SRange(0., 33.4), "Bidirectional", strict_severity},
            {LaneId("1_2_1"), SRange(0., 33.4), "AgainstS", strict_severity},
-           {LaneId("1_2_-1"), SRange(0., 33.4), "Undefined", strict_severity},
```

Comment:
We probably have to change something at RuleRegistry as Undefined is not a valid state anymore

Reply:
Removed one test setup line which was not doing anything. I didn't find anything else.